gradle/versions.gradle

```diff
@@ -12,7 +12,7 @@ ext {
     slf4jVersion = "1.7.25"
     jsonPathVersion = "2.4.0"
     junitVersion = "4.12"
-    explorerApiCommonVersion = "1.0.0"
+    explorerApiCommonVersion = "1.0.2-PR-54-20200402.151153-6"
```

Comment:
I'm guessing this will be changed to 1.1 after the explorer-api-common has been merged?
Same for the test version below

Reply:
Correct!